erpnext/selling/doctype/customer/customer.py

```
@@ -58,7 +58,7 @@ class Customer(TransactionBase):
 		customer_pos_id: DF.Data | None
 		customer_primary_address: DF.Link | None
 		customer_primary_contact: DF.Link | None
-		customer_type: DF.Literal["Company", "Individual", "Partnership"]
+		customer_type: DF.Literal["Personne Physique", "Personne Morale", "Personne physique Commerçante", "Profession Libérale", "Ambassade et Organisation Internationale"]
```

Ruff (0.15.10)

[error] Syntax error in forward annotation: Unexpected token at the end of an expression

(F722)